addons/stock_account/__init__.py

```diff
@@ -16,7 +16,8 @@ def _configure_journals(cr, registry):
 
     for company_id in company_ids:
         # Check if property exists for stock account journal exists
-        properties = env['ir.property'].search([
+        # TODO use native ir.property methods?
+        properties = env['ir.property'].sudo().search([
             ('name', '=', 'property_stock_journal'),
             ('company_id', '=', company_id.id)])
```

Comment:
What worries me in this code is that there is no condition on the model's name.
It is correct provided nobody adds a field `property_stock_journal` on another model!